ui/webui/src/helpers/product.js

```diff
@@ -31,3 +31,7 @@ export const getOsReleaseByKey = (key) => {
             )
             .finally(confFile.close);
 };
+
+export const getAnacondaVersion = () => {
```

Comment:
Can you please do the version parsing here, and return the correct value?

Reply:
@M4rtinK doesn't the dbus API return the Anaconda version?

Reply:
hi @jelly @M4rtinK just wondering if indeed there is an api that returns the anaconda version or if i should keep the` product.js` file to use `cockpit.spawn(["anaconda", "--version"]);`